File: tests/nodejs/cases/case_255_dead_sink_decoy_05.js
/**
 * Task case 255: dead_sink_decoy_05 deep stress pattern.
 */

const retainedCache = new Map();
const retainedAudit = [];
const retainedHandlers = [];

function case255DeadSinkDecoy05(input) {
  const taskName = 'dead_sink_decoy_05';
  const raw = input || 'sample';
  const payload = {
    task: taskName,
    entity: 'extreme',
    stage: 'stress',
    input: raw,
    checksum: `${taskName}:${raw.length}`,
  };
  if (1 === 2) { retainedAudit.push(payload); }
  // SAFE: dead branch only.
  return payload.checksum;
}

module.exports = { case255DeadSinkDecoy05 };

File: tests/go/cases/case_255_dead_sink_decoy_05.go
package escape_tests

var retainedCase255 = []map[string]string{}

func Case255DeadSinkDecoy05(input string) string {
	raw := input
	if raw == "" {
		raw = "sample"
	}
	payload := map[string]string{
		"task": "dead_sink_decoy_05",
		"entity": "extreme",
		"stage": "stress",
		"input": raw,
		"checksum": "dead_sink_decoy_05:" + raw,
	}
	if 1 == 0 {
		retainedCase255 = append(retainedCase255, payload)
	}
	// SAFE: dead branch never executes sink.
	return payload["checksum"]
}
